docgen/src/guide/Migration_guide_v5.md

```diff
@@ -187,7 +187,7 @@ No change.
 
 #### Behaviour
 
-Instead of displaying the widget as `category: One ✕ Two ✕`, it now displays like this: `One ✕` `Two ✕`.
+No change.
```

Comment:
aren't the classes still different?

Reply:
Right 😅

Reply:
Actually in the migration guide there is nothing to add, since we list only the changes of the classnames. But I updated the widget file with the missing classnames.